game/libda/fft.hpp

```diff
@@ -9,11 +9,9 @@
 #include <cstddef>
 #include <vector>
 
-#ifndef M_PI
-#define M_PI 3.141592653589793
-#endif
+extern const double m_pi;
 
-#define M_TAU (2.0 * M_PI)
```

Comment:
yeah, macros are ugly and give really weird error codes since the preprocessor fills them in before any error is thrown, so it's best to get rid of them!